### ISSUE TITLE
This repo may be mis-reporting histogram references

### ISSUE THREAD

**Issue by justinmchase** [mafintosh/why-is-node-running#58]
This repo appears to be mis-reporting [`histogram.enable()`](https://github.com/siimon/prom-client/blob/master/lib/metrics/eventLoopLag.js#L47-L50) as a lingering reference:

Related to https://github.com/siimon/prom-client/issues/494

```
# ELDHISTOGRAM
node:internal/async_hooks:201                                                   
node:internal/perf/event_loop_delay:81                                          
node:internal/perf/event_loop_delay:78                                          
/var/app/example/node_modules/prom-client/lib/metrics/eventLoopLag.js:47 - const histogram = perf_hooks.monitorEventLoopDelay({
/var/app/example/node_modules/prom-client/lib/defaultMetrics.js:43       - metric(config.register, config);
```

This was reported upstream to nodejs https://github.com/nodejs/node/issues/42132 and they concluded histograms should not be showing here since they are `unref`'d at the runtime level. Therefore we suspect that it may be a false positive here in this repo.

I see that you have a number of other types blocklisted here:
https://github.com/mafintosh/why-is-node-running/blob/24fb4c878753390a05d00959e6173d0d3c31fddd/index.js#L8-L11

I'm wondering if histogram's should be added here as well?

**Comment by twpol** [comment]
Might be fixed by #60?

**Comment by RaisinTen** [comment]
https://github.com/mafintosh/why-is-node-running/pull/60 has just landed and it has been released in v2.2.2. Could you please check if this is still an issue?

**Comment by jonkoops** [comment]
I am unable to reproduce this in the latest version, and looking at the discussion here I think we can go ahead and close this issue.
